types/hooks/screens/useMessageHook.types.ts

```diff
@@ -32,6 +32,7 @@ export interface UseMessageHook {
     data: any,
     type: 'ANONYMOUS' | 'SIGNED'
   ) => void;
+  onOpenMediaPreview: (medias: any, index: number, navigation: any) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onOpenMediaPreview` function has a parameter `medias` which is of type `any`. Using `any` type can lead to potential issues as it bypasses type checking. It would be better to define an interface for `medias` if possible, or at least use a more specific type.

```diff
- onOpenMediaPreview: (medias: any, index: number, navigation: any) => void;
+ onOpenMediaPreview: (medias: Media[], index: number, navigation: NavigationType) => void;
```
In the above code, `Media` and `NavigationType` are hypothetical types that you might want to define based on your application's requirements.


<!-- This is an auto-generated comment by OSS CodeRabbit -->